trade-finance-manager-ui/server/api.js

```diff
@@ -1230,6 +1230,23 @@ const addFeesToAnExistingPayment = async (reportId, feeRecordIds, paymentIds, us
   return response.data;
 };
 
+/**
+ * @param {string} dealId - The deal ID
+ * @param {Partial<import('@ukef/dtfs2-common').TfmDealCancellation>} cancellationUpdate - The deal cancellation update object
+ * @param {string} userToken - The user token
+ * @returns {Partial<import('@ukef/dtfs2-common').TfmDealCancellation>} - The deal cancellation update object
```

Comment:
Also should be an update result (though thats a mongodb type which we cant import in tfm-ui so maybe want something else - honestly could just be void right?)

Reply:
Yep good point, we don't need the returned object so have changed it to not return anything / updated docstrings to `@returns {Promise<void>}`